packages/jaeger-ui/src/components/Monitor/EmptyState/index.tsx

```diff
@@ -59,7 +59,7 @@ export default class MonitorATMEmptyState extends React.PureComponent {
             )}
             {this.config.alert && (
               <Row justify="center">
```

Comment:
is this correct that we have a row inside a row?

Reply:
As far as I am seeing, it's giving the expected result. Not sure 100% though, as there is not a strict guide about nested row and cols.